build/controllers/TranslationController.php

```diff
@@ -126,11 +126,11 @@ public function highlightDiff($diff)
     {
         $lines = explode("\n", $diff);
         foreach ($lines as $key => $val) {
-            if (mb_substr($val, 0, 1, 'utf-8') === '@') {
+            if (strpos($val, '@') === 0) {
```

Comment:
Should it be `mb_strpos`?

Reply:
I don't think so. `strpos` is binary-safe and we are looking for ASCII character.